webrtc.html

```diff
@@ -3777,7 +3777,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       SHA-256 hash.</p>
       <div>
         <pre class="idl">partial interface RTCPeerConnection {
-    static Promise&lt;RTCCertificate&gt; generateCertificate (AlgorithmIdentifier keygenAlgorithm);
+    static Promise&lt;RTCCertificate&gt; generateCertificate (RTCGenerateCertificateOptions keygenAlgorithm);
```

Comment:
It currently takes an AlgorithmIdentifier. Any string that would be accepted by WebCrypto's normalizeAlgorithm[1] is an AlgorithmIdentifier in Chromium. According to a comment in the code, a String is treated the same as a Dictionary with a "name" attribute and nothing else.

However, we reject any AlgorithmIdentifier that doesn't have expected non-name parameters such as modulusLength, publicExponent, hash and namedCurve, so that depends on if the default values for these align with parameters we accept in generateCertificate.

... And, I just checked, neither "RSA" or "ECDSA" are accepted Strings for generateCertificate. Perfect, that means this should be a safe change.

[1] https://cs.chromium.org/chromium/src/third_party/WebKit/Source/modules/crypto/NormalizeAlgorithm.h?sq=package:chromium&dr&rcl=1477293892&l=66